helm/pv-cleaner-operator-chart/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pv-cleaner-operator-configmap
+  namespace: giantswarm
+data:
+  config.yml: |
+    server:
+      listen:
+        address: 'http://0.0.0.0:8000'
+    service:
+      kubernetes:
+        address: ''
```

Comment:
Out of curiosity, do we need to include these empty values if `inCluster` is true?

Reply:
Didn't check, but that is the same for all configmap operators templates. So, even if it matters - we will need to update everywhere

Reply:
ack